plugin/postgres/plugin.go

```diff
@@ -154,8 +163,21 @@ func (p PostgresPlugin) Backup(endpoint ShieldEndpoint) error {
 	}
 
 	setupEnvironmentVariables(pg)
-	cmd := fmt.Sprintf("%s/pg_dumpall -c --no-password", pg.Bin)
-	DEBUG("Executing: `%s`", cmd)
+
+	// If user specified specific database to backup
+	db_name := pg.Database
```

Comment:
Is there a compelling reason to use a temporary variable?  I see no transformations on the `db_name` copy of `pg.Database`...